handbook/company/product-groups.md

```diff
@@ -742,7 +735,16 @@ This meeting is to disseminate engineering-wide announcements, promote cohesion
   - Everyone is welcome to present on a technical topic. Add your name and tech talk subject in the agenda doc included in the Eng Together calendar event.
 - Social
   - Structured and/or unstructured social activities
+ 
+### New customer promise(s)
 
+This meeting happens before Fleet commits to one or more new customer promises. It's meant to streamline communication and encourage getting the best product decisions.
```

Comment:
I think we are missing who is the DRI for scheduling this meeting?

Reply:
Good point. I think it makes sense for @alexmitchelliii to be DRI for scheduling. Alex knows first when we want to add something to an order form.

I updated the PR to reflect this: https://github.com/fleetdm/fleet/pull/22064/files#diff-c99d12c3af50c0c2aca2b9ef7597c02ccfe87678291956ff0b2e83d63978ea38R741